lib/components/home/artwork_rails/artwork_rail_header.js

```diff
@@ -3,25 +3,51 @@
 
 import Relay from 'react-relay'
 import React from 'react'
-import { StyleSheet, Dimensions, View, Text } from 'react-native'
+import { StyleSheet, Dimensions, View, Text, TouchableWithoutFeedback } from 'react-native'
 
 import SerifText from '../../text/serif'
 import colors from '../../../../data/colors'
 import Button from '../../buttons/inverted_button'
+import SwitchBoard from '../../../modules/switch_board'
+import fragments from './relay_fragments'
```

Comment:
These were cluttering up this component quite a bit so I moved them into their own file. A lot of the routing here (and therefore these fragments) is identical to the routing for the View All button in the main `ArtworkRail` component. I'm currently thinking about nice ways to abstract the routing and fragments so they can be used by both components.